src/libs/reportUtils.js

```diff
@@ -152,6 +152,20 @@ function isConciergeChatReport(report) {
         && report.participants[0] === CONST.EMAIL.CONCIERGE;
 }
 
+/**
+ * Whether a login is system email
+ *
+ * @param {String} login - user email
+ * @return {Boolean}
+ */
+function isSystemUser(login) {
```

Comment:
NAB but `reportUtils` feels like a weird place for this. Not sure I have a better suggestion for now unless we create `userUtils` or something?